app/src/main/java/org/simple/clinic/ClinicApp.kt

```diff
@@ -43,13 +44,15 @@ abstract class ClinicApp : Application(), CameraXConfig.Provider {
 
   protected open val crashReporterSinks = emptyList<CrashReporter.Sink>()
 
-  @SuppressLint("RestrictedApi")
+  @SuppressLint("RestrictedApi", "UnsafeOptInUsageWarning")
```

Comment:
Where can I find the definition of `UnsafeOptInUsageWarning` id that we're asking lint to ignore?

Reply:
http://googlesamples.github.io/android-custom-lint-rules/checks/UnsafeOptInUsageWarning.md.html